packages/features/src/onboarding/views/mnemonic-input.tsx

```diff
@@ -67,7 +71,7 @@ export const MnemonicInputView = () => {
         restoreArgs,
         Mina.Networks.BERKELEY,
```

Comment:
A reminder for both of us to make sure we need to comb through the application for hard-coded variables 💡 

I've made an issue for us to tackle later: https://github.com/palladians/pallad/issues/114